source/_components/remember_the_milk.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "Remember The Milk"
+description: "Instructions on how to use Remember The Milk with Home Assistant."
+date: 2017-10-10 10:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.23
+---
+
+The `Remember The Milk` component allows you to create tasks in [Remember The Milk](https://www.rememberthemilk.com) from Home Assistant. You can use this if you want Home Assistant to send you a task that you should not forget, e.g. water the plants.
+
+# Setup
+
+The setup consists of two steps: getting an API key and registering your account
+
+## step 1: API key
+
+To be able to use this component, you need a Remember The Milk account and you need to apply for your own [API key](https://www.rememberthemilk.com/services/api/keys.rtm). With the API key you will also receive your personal `shared secret`. Both of them need to be stored in your Home Assistant configuration:
+
+```yaml
+# Example configuration.yaml entry
+
+remember_the_milk:
+  your_rtm_account:
+    api_key: <your secret api key goes here>
+    shared_secret: <your secret shared secret goes here>
+
+```
+
```

Comment:
Please describe the [configuration options](https://home-assistant.io/developers/documentation/create_page/#configuration).

Reply:
OK